shared/config.gradle

```diff
@@ -43,6 +43,8 @@ nativeUtils.platformConfigs.each {
 nativeUtils.platformConfigs.each {
   if (it.name.contains('linux')) {
     it.cppCompiler.debugArgs.add("-gz=zlib")
+    // Suppress warning in OpenCV 4.10 from GCC 15
+    it.cppCompiler.args.add("-Wno-overloaded-virtual")
```

Comment:
Do we really want to completely disable the warning, versus leaving the warning enabled and just not erroring on it?  (e.g. `-Wno-error=no-overloaded-virtual`)